rules-reviewed/jboss-eap/websphere/tests/WebSphereMQ.windup.test.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0"?>
+<ruletest xmlns="http://windup.jboss.org/schema/jboss-ruleset">
+    <testDataPath>samples/WebSphereMQ</testDataPath>
+    <rulePath>../WebSphereMQ.windup.xml</rulePath>
+    <ruleset>
+        <metadata></metadata>
+        <rules>
+            <rule>
```

Comment:
But what happens when you have more than one test for a rule? E.g. In the scenario where I suggestedd parameterization of the rules in the JMS ruleset in this PR? One parameterized rule could match a ton of things. We might have a few tests just for that one rule. (In that case, we could add a suffix to the test rule ID:

ruleset-id-02000-1
ruleset-id-02000-2
ruleset-id-02000-3

... etc